src/celengine/refcounted.cpp

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
this file is not required, it's effectively empty

Reply:
As above...